app/controllers/candidates/registrations/placement_preferences_controller.rb

```diff
@@ -42,10 +43,21 @@ def set_school
         @school = Candidates::School.find(params[:school_id])
       end
 
+      def set_placement_dates
+        if @school.fixed_dates?
+          @placement_dates = @school
+            .school_profile
```

Comment:
I think we should get the bookings_placement_dates directly from the school rather than going throught the school_profile. I know this is getting changed in a future PR, just leaving this comment here for clarity